src/js/mixins/evented.js

```diff
@@ -499,6 +500,11 @@ function evented(target, options = {}) {
   // When any evented object is disposed, it removes all its listeners.
   target.on('dispose', () => {
     target.off();
+    [target, target.el_, target.eventBusEl_].forEach(function(val) {
```

Comment:
dispose of any and all dom data here, previously we did this in component.js, but I think we need it for any evented mixin object.